sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_service_groundedness/_service_groundedness.py

```diff
@@ -0,0 +1,150 @@
+from azure.ai.evaluation._evaluators._common import RaiServiceEvaluatorBase
+
+
+@experimental
+class GroundednessProEvaluator(RaiServiceEvaluatorBase):
+    """
+    Initialize a Groundedness Pro evaluator for determine if the response is grounded
+    in the query and context.
+
+    If this evaluator is supplied to the `evaluate` function, the aggregated metric
+    for the groundedness pro label will be "groundedness_pro_passing_rate".
+
+    :param credential: The credential for connecting to Azure AI project. Required
+    :type credential: ~azure.core.credentials.TokenCredential
+    :param azure_ai_project: The scope of the Azure AI project.
+        It contains subscription id, resource group, and project name.
+    :type azure_ai_project: ~azure.ai.evaluation.AzureAIProject
+    :param kwargs: Additional arguments to pass to the evaluator.
+    :type kwargs: Any
+
+    **Usage**
+
+    .. code-block:: python
+
+        azure_ai_project = {
+            "subscription_id": "<subscription_id>",
+            "resource_group_name": "<resource_group_name>",
+            "project_name": "<project_name>",
+        }
+        credential = DefaultAzureCredential()
+
+        eval_fn = GroundednessProEvaluator(azure_ai_project, credential)
+        result = eval_fn(query="What's the capital of France", response="Paris", context="Paris.")
+
+    **Output format**
+
+    .. code-block:: python
+
+        {
+            "groundedness_pro_label": True,
+            "reason": "'All Contents are grounded"
+        }
+
+    **Usage with conversation input**
+
+    .. code-block:: python
+
+        azure_ai_project = {
+            "subscription_id": "<subscription_id>",
+            "resource_group_name": "<resource_group_name>",
+            "project_name": "<project_name>",
+        }
+        credential = DefaultAzureCredential()
+
+        eval_fn = GroundednessProEvaluator(azure_ai_project, credential)
+        conversation = {
+            "messages": [
+                {"role": "user", "content": "What is the capital of France?"},
+                {"role": "assistant", "content": "Paris.", "context": "Paris."}
+                {"role": "user", "content": "What is the capital of Germany?"},
+                {"role": "assistant", "content": "Berlin.", "context": "Berlin."}
+            ]
+        }
+        result = eval_fn(conversation=conversation)
+
+    **Output format**
+
+    .. code-block:: python
+
+            {
+                "groundedness_pro_label": 1.0,
+                "evaluation_per_turn": {
+                    "groundedness_pro_label": [True, True],
+                    "groundedness_pro_reason": ["All contents are grounded", "All contents are grounded"]
+                }
+            }
+    """
+
+    @override
+    def __init__(
+        self,
+        credential,
+        azure_ai_project,
+        **kwargs,
+    ):
+        self._passing_score = 3  # TODO update once the binarization PR is merged
```

Comment:
@posaninagendra, in order to parity with AACS groundedness, any ungrounded content detected will make its AACS binary output ungroundedDetected to be True. 
1. isn't ungroundedDetected part of the service call output? 
2. if not, and if SDK only receives ungroundedPercentage as the output, then to match the logic for ungroudnedDetected, this self._passing_score should be 5, right?

Reply:
For reference, here is the sample output on the [AACS doc](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/quickstart-groundedness?tabs=curl#check-groundedness-with-reasoning): 

`{
  "ungroundedDetected": true,
  "ungroundedPercentage": 1,
  "ungroundedDetails": [
    {
      "text": "12/hour.",
      "offset": {
        "utf8": 0,
        "utf16": 0,
        "codePoint": 0
      },
      "length": {
        "utf8": 8,
        "utf16": 8,
        "codePoint": 8
      },
      "reason": "None. The premise mentions a pay of \"10/hour\" but does not mention \"12/hour.\" It's neutral. "
    }
  ]
}`